statics/js/components/inject-form.js

```diff
@@ -22,10 +22,13 @@ Vue.component('inject-form', {
                        id="inject-src"\
                        attr="id"\
                        v-model="node1"></node-selector>\
+        <input id="src-ip" class="form-control input-sm inject-target" v-model="srcIP" placeholder="Src IP"/>\
         <node-selector placeholder="To"\
+                       class="inject-target"\
                        attr="id"\
                        id="inject-dst"\
                        v-model="node2"></node-selector>\
+        <input id="dst-ip" class="form-control input-sm" v-model="dstIP" placeholder="Dst IP"/>\
```

Comment:
I uploaded a draft in order to make it a bit easier to understand (IHMO) https://da.gd/jHPbO. We could fill the IP text input with the auto selected IP if present and let the user override it.

Reply:
with "form-group" class, we can't have 100% width. so I will do the require modification with input-group and submit the code.

Reply:
I think that is not a big issue if the input doesn't reach the right end of the screen (IPs are short) but if you can have the IP (label) and the input text wider it will be perfect.